test/e2e/testdata/fail-forward/replaces/example-operator.v0.3.0.yaml

```diff
@@ -0,0 +1,26 @@
+---
+schema: olm.package
+name: packageA
+defaultChannel: stable
+---
+schema: olm.channel
+package: packageA
+name: stable
+entries:
+  - name: example-operator.v0.3.0
+    replaces: example-operator.v0.2.0
+---
+schema: olm.bundle
+name: example-operator.v0.3.0
+package: packageA
+image: quay.io/joelanford/example-operator-bundle:0.3.0
```

Comment:
should we use an "official" repository rather than Joe's?

Reply:
Just saw you have an action item for that in the description. I think Ankita's is being used too (but I'm sure you know - just in case)

Reply:
Just saw you have an action item for that in the description. I think Ankita's is being used too (but I'm sure you know - just in case)